svg.js.d.ts

```diff
@@ -80,14 +80,14 @@ declare module "@svgdotjs/svg.js" {
 
     }
 
-    let easing: {
-        '-'(pos: number): number;
-        '<>'(pos: number): number;
-        '>'(pos: number): number;
-        '<'(pos: number): number;
-        bezier(x1: number, y1: number, x2: number, y2: number): (t: number) => number;
-        steps(steps: number, stepPosition?: "jump-start"|"jump-end"|"jump-none"|"jump-both"|"start"|"end"): (t: number, beforeFlag?: boolean) => number;
-    }
+    // let easing: {

```

Comment:
This easing object is also exported so it should actually be an export

Reply:
Added back...

Reply:
So let me ask a consultant-type question: what is the use case for ForeignObject, not in theory but in practice. What hole does it help fill?

Right now is will accept basically anything and its dog; maybe it doesn't need to. Perhaps _all_ you need to support is HTMLElement. What would be lost if that were the case?

Reply:
ForeignObject can be filled with basically any Node you want. It is for content which is NOT svg. So it can be html, mathml, xml - whatever you want.
Since svg.js also works in standalone svgs and on the server, it basically needs to support anything which can be intepretet by any interpreter which exist or might exist.

But were is this question coming from? Because of the `add()` method?